### PR TITLE
URL: URL.canParse()

### PR DESCRIPTION
**Pull request by annevk**
For https://github.com/whatwg/url/pull/763.

### CODE REVIEW

File: url/url-statics-canparse.any.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "base": undefined,
+    "expected": false
+  },
+  {
+    "url": "a",
+    "base": "https://b/",
+    "expected": true
+  }
+].forEach(({ url, base, expected }) => {
+  test(t => {
```
Comment (ricea) [inline url/url-statics-canparse.any.js:39]: Nit: `t` is unused.

**Review by ricea — APPROVED** [review]
lgtm

**Comment by zloirock** [comment]
[One more case.](https://github.com/nodejs/node/issues/47505)

**Comment by annevk** [comment]
Create a PR? Happy to review next week.

**Comment by zloirock** [comment]
[Similar Deno bug.](https://github.com/denoland/deno/issues/18893)

**Pull request merged**